src/FlightMap/MapItems/VehicleMapItem.qml

```diff
@@ -41,7 +42,7 @@ MapQuickItem {
         id:         vehicleItem
         width:      vehicleIcon.width
         height:     vehicleIcon.height
-        opacity:    vehicle ? (vehicle.active ? 1.0 : 0.5) : 1.0
+        opacity:    (vehicle && _activeVehicle) ? ((vehicle === _activeVehicle) ? 1.0 : 0.5) : 1.0
```

Comment:
Yes, good bug find. I think this would make the code a little simpler/clearer as to what needs to happen with respect to ADSB vehicle as well as multiple vehicles with only one being active:

`opacity:    _adsbVehicle ||  vehicle === _activeVehicle ? 1.0 : 0.5`

Reply:
That's much simpler and I didn't even think about ADSB vehicles. Done in latest commit.